docs/designers-developers/developers/block-api/versions.md

```diff
@@ -0,0 +1,12 @@
+# Block API Versions
+
+This document lists the changes made between the different API versions.
+
+## Version 2 (>= WordPress 5.6)
+
+- Moves the responsibility to render the block element wrapper in the editor to the block author using the `useBlockProps()` hook.
+- Generates classnames and styles are not added automatically to the saved markup for static blocks, block authors are required to explicitely use `useBlockProps.save()` in their `save` functions to retrieve the generated classes and styles to apply on the block wrapper.
```

Comment:
```suggestion
- The generated classnames and styles are no longer added automatically to the saved markup for static blocks.
- To render the block element wrapper, the block author must use the `useBlockProps()` hook.
- To retrieve the generated classes and styles, the block author must explicitly use `useBlockProps.save()` and add to their block wrapper.
```

Rewording a little. I think we could also link or reference a simple block that illustrates the change.

Reply:
The first point and the last point are the same for me. I think we should put them together because both are about the `save` function.

I think we should  clarify  that the  second point is about the `edit` function (or the editor representation of the block)

Reply:
What about:
```md
- To render the block element wrapper for the block's `edit` implementation, the block author must use the `useBlockProps()` hook.
- The generated class names and styles are no longer added automatically to the saved markup for static blocks when `save` is processed. To include them, the block author must explicitly use `useBlockProps.save()` and add to their block wrapper.
```